src/org/labkey/test/components/domain/DomainFieldRow.java

```diff
@@ -916,6 +975,40 @@ public DomainFieldRow deleteTextChoiceValue(String value)
         return this;
     }
 
+    // Calculations field settings
+
+    public DomainFieldRow setValueExpression(String expression)
+    {
+        if (!isExpanded())
+            expand();
+
+        getWrapper().setFormElement(elementCache().expressionInput, expression);
+        return this;
+    }
+
+    public String getValueExpressionStatusMessage()
+    {
+        // Need to remove focus from the expression field to have the updated status message show up.
+        getWrapper().fireEvent(elementCache().expressionInput, WebDriverWrapper.SeleniumEvent.blur);
+        WebDriverWrapper.sleep(500);
+
+        String statusMsg = "";
+        if(Boolean.TRUE.equals(waitFor(elementCache().expressionStatusMsg::isDisplayed, 1_000)))
```

Comment:
No need to do `TRUE.equals` here, `WebElement.isDisplayed()` won't ever return `null`.
```suggestion
        if(waitFor(elementCache().expressionStatusMsg::isDisplayed, 1_000))
```

Reply:
Changed.